src/index.html

```diff
@@ -1,12 +1,442 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="Page icon">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-and-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <ul class="catalog page__catalog">
+        <li class="catalog__item" data-qa="card">
+          <section class="card">
```

Comment:
useless container, just create a mix
```suggestion
          <section class="card catalog__item">
```